model-optimizer/extensions/front/Inpaint.py

```diff
@@ -0,0 +1,82 @@
+"""
```

Comment:
Why this file is here?

Reply:
It was added by mistake. Now it is removed.